drivers/gles3/shaders/sky.glsl

```diff
@@ -28,7 +28,7 @@ void main() {
 	// We're doing clockwise culling so flip the order
 	uv_interp = vec2(vertex_attrib.x, vertex_attrib.y * -1.0);
 #endif
-	gl_Position = vec4(uv_interp, 1.0, 1.0);
+	gl_Position = vec4(uv_interp, -1.0, 1.0);
```

Comment:
Perhaps this change should only happen within the `#ifdef USE_INVERTED_Y`?

Reply:
Hm, no, this isn't the issue. I tried reverting the change, and it messed up depth - I see the things behind rather than in front.